src/objects/index.ts

```diff
@@ -1,15 +1,21 @@
-export type User = Agent | Customer;
+import { 
+  Customer as CustomerCustomer,
+  Agent as CustomerAgent,
+  Statistics as CustomerStatistics,
+  Geolocation as CustomerGeolocation,
+  LastPage as CustomerLastPage,
+  Visit as CustomerVisit,
+ } from "../agent/structures";
+ import {
+  Customer as AgentCustomer,
+  Agent as AgentAgent,
+ } from "../customer/structures";
+
+export type Agent = AgentAgent | AgentCustomer;
```

Comment:
Also, shouldn't this be
```suggestion
export type Agent = AgentAgent | CustomerAgent;
```
?

Reply:
https://github.com/livechat/lc-sdk-js/pull/65#discussion_r907209380